src/test/suite/didactUriCompletionItemProvider.test.ts

```diff
@@ -74,17 +77,14 @@ suite("Didact URI completion provider tests", function () {
 
 		suite('walk through each provided completion', () => {
 			listOfCompletions.forEach(function(stringToTest: string){
-				before( async () => {
-					await delay(2000);
-				});
-
 				afterEach( async () => {
 					await removeFilesAndFolders(testWorkspace, foldersAndFilesToRemove);
 				});
-			
+
 				test(`test provided completion "${stringToTest}"`, async () => {
 					await executeCompletionTest(stringToTest, expected);
-				});
+					}).timeout(COMPLETION_TIMEOUT);
```

Comment:
I do not understand the link between removing the delay in before method and adding a timeout. Can you explain how it is influencing it?

Reply:
The test timeout is giving a bit more space for the "executeCompletionTest" to finish -- and the before delay was simply to wait for the file removal to be complete I believe. @lhein made the suggestion

Reply:
so means that we are no more waiting for the file removal?

Reply:
no, we're still awaiting for the file removal (there are awaits used in removeFilesAndFolders and in the afterEach call) - but the timeout seems to be giving the extra little space the before was giving us without adding an extra delay.

Reply:
the timeout is not specifically giving extra time between tests.
it sounds more like the before with delay had no utility and the test for completion needed an higher timeout.